Model/Page.php

```diff
@@ -1,5 +1,27 @@
 <?php
 
-class Training_Cms_Model_Page extends Mage_Core_Model_Abstract{
+class Training_Cms_Model_Page extends Mage_Core_Model_Abstract
+{
+
+    /**
```

Comment:
This PHPDoc is obvious. If there are no tools that are parsing PHPDoc block then we don't need blocks that duplicate information that we can get from method's title or parameter's name. Please remove PHPDoc blocks that don't add any new information.